sql/dd/impl/upgrade/server.cc

```diff
@@ -470,6 +470,67 @@ static void create_upgrade_file() {
 
 }  // namespace
 
+#ifdef WITH_WSREP
+
+bool pxc_fix_mysql_tables(THD *thd) {
+  const char section_start[] = "#! PXC_SECTION::START";
+  const char section_end[] = "#! PXC_SECTION::END";
+  int section_start_len = strlen(section_start);
```

Comment:
For both: could be `const LEX_CSTRING section_start = (STRING_WITH_LEN("...");`

Or at least the lens could be `const size_t` instead of `int`.

Reply:
code modified to use LEX_CSTRING